src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlReflectionImporter.cs

```diff
@@ -1291,6 +1321,7 @@ private MembersMapping ImportMembersMapping(XmlReflectionMember[] xmlReflectionM
             return members;
         }
 
+        [RequiresUnreferencedCode("calls .ctor")]
```

Comment:
This one is odd - calls what ctor?

Reply:
For both places, I've changed the comment to point to the right method that causes this one to be unsafe.